doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1538,9 +1538,14 @@ numbers (see :ref:`datatypes`).
 Numeral notations
 ~~~~~~~~~~~~~~~~~
 
-.. cmd:: Numeral Notation @qualid @qualid__parse @qualid__print : @scope_name {? @numeral_modifier }
+.. cmd:: Numeral Notation @qualid {? @numeral_via } @qualid__parse @qualid__print : @scope_name {? @numeral_modifier }
    :name: Numeral Notation
 
+   .. insertprodn numeral_via numeral_via
+
+   .. prodn::
+      numeral_via ::= via @qualid__ind using ( {+, @qualid @qualid } )
+
    .. insertprodn numeral_modifier numeral_modifier
```

Comment:
You only need a single `insertprodn` and `prodn` here.  In `orderedGrammar`, move the definition for `numeral_via` to be just before `numeral_modifier` and update the `insertprodn` arguments here.  Then rerun `make doc_gram_rsts`.

Reply:
Thanks